package proto_ff;

import "yd_fieldoptions.proto";


message E_FindtreasureFindtreasure
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_functionUnlock = 3[(yd_fieldoptions.field_cname) = "功能开放"];
	optional int64 m_costID = 4[(yd_fieldoptions.field_cname) = "道具id"];
	optional string m_costNum = 5[(yd_fieldoptions.field_cname) = "抽奖消耗", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_ContItem = 6[(yd_fieldoptions.field_cname) = "特殊连抽道具"];
	optional string m_poolGroupId = 7[(yd_fieldoptions.field_cname) = "奖池id", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_loop = 8[(yd_fieldoptions.field_cname) = "第几个奖池后为循环奖池"];
	optional int32 m_poolTime = 9[(yd_fieldoptions.field_cname) = "奖池轮换时间"];
	optional int32 m_totalGroupID = 10[(yd_fieldoptions.field_cname) = "累计次数奖励id"];
	optional int32 m_activeGroupID = 11[(yd_fieldoptions.field_cname) = "活动id"];
	optional int32 m_protectTime = 12[(yd_fieldoptions.field_cname) = "保底奖池次数"];
	optional int32 m_addItem = 13[(yd_fieldoptions.field_cname) = "抽奖额外奖励ID"];
	optional int32 m_addNum = 14[(yd_fieldoptions.field_cname) = "抽奖额外奖励数量"];
	optional int32 m_shopID = 15[(yd_fieldoptions.field_cname) = "商城Id"];
	optional int32 m_storehouseType = 16[(yd_fieldoptions.field_cname) = "仓库类型"];
}


message Sheet_FindtreasureFindtreasure
{
	repeated E_FindtreasureFindtreasure E_FindtreasureFindtreasure_List = 1[(yd_fieldoptions.field_arysize)=16];
}

message E_FindtreasureFindpool
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_groupID = 2[(yd_fieldoptions.field_cname) = "奖池组ID"];
	optional int32 m_showOrder = 3[(yd_fieldoptions.field_cname) = "客户端展示"];
	optional int64 m_itemid = 4[(yd_fieldoptions.field_cname) = "物品"];
	optional int32 m_num = 5[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_prof = 6[(yd_fieldoptions.field_cname) = "职业"];
	optional int32 m_weights = 7[(yd_fieldoptions.field_cname) = "掉落权重"];
	optional int32 m_proWeights = 8[(yd_fieldoptions.field_cname) = "保底掉落权重"];
	optional int32 m_poolLV = 9[(yd_fieldoptions.field_cname) = "奖池等级"];
	optional int32 m_Record = 10[(yd_fieldoptions.field_cname) = "奖励记录稀有度"];
}


message Sheet_FindtreasureFindpool
{
	repeated E_FindtreasureFindpool E_FindtreasureFindpool_List = 1[(yd_fieldoptions.field_arysize)=256];
}

message E_FindtreasureConstant
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_dragonSeverTime = 2[(yd_fieldoptions.field_cname) = "真龙秘宝开服开启常规刷新时间"];
	optional int32 m_godSeverTime = 3[(yd_fieldoptions.field_cname) = "神机更换奖池时间"];
	optional string m_alchemySucces = 4[(yd_fieldoptions.field_cname) = "炼金工坊各等级的晋升成功率", (yd_fieldoptions.field_bufsize) = 64];
}


message Sheet_FindtreasureConstant
{
	repeated E_FindtreasureConstant E_FindtreasureConstant_List = 1[(yd_fieldoptions.field_arysize)=2];
}

message E_FindtreasureTotal
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_groupid = 2[(yd_fieldoptions.field_cname) = "组id"];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_itemID = 4[(yd_fieldoptions.field_cname) = "奖励道具id"];
	optional int32 m_num = 5[(yd_fieldoptions.field_cname) = "奖励道具数量"];
	repeated int32 m_arg = 6[(yd_fieldoptions.field_cname) = "参数", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_FindtreasureTotal
{
	repeated E_FindtreasureTotal E_FindtreasureTotal_List = 1[(yd_fieldoptions.field_arysize)=64];
}

message E_FindtreasureActive
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_groupid = 2[(yd_fieldoptions.field_cname) = "组id"];
	optional int32 m_opentime = 3[(yd_fieldoptions.field_cname) = "活动开启时间"];
	optional int32 m_durTime = 4[(yd_fieldoptions.field_cname) = "持续时间"];
	optional int32 m_closeTime = 5[(yd_fieldoptions.field_cname) = "关闭时间"];
	optional int32 m_type = 6[(yd_fieldoptions.field_cname) = "活动类型"];
	optional int32 m_typeGroupId = 7[(yd_fieldoptions.field_cname) = "活动组id"];
}


message Sheet_FindtreasureActive
{
	repeated E_FindtreasureActive E_FindtreasureActive_List = 1[(yd_fieldoptions.field_arysize)=8];
}

message E_FindtreasureActiveup
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_groupID = 2[(yd_fieldoptions.field_cname) = "组id"];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "分类"];
	optional int32 m_itemid = 4[(yd_fieldoptions.field_cname) = "物品id"];
	optional int32 m_proArg = 5[(yd_fieldoptions.field_cname) = "调整系数"];
}


message Sheet_FindtreasureActiveup
{
	repeated E_FindtreasureActiveup E_FindtreasureActiveup_List = 1[(yd_fieldoptions.field_arysize)=32];
}

message E_FindtreasureActivegoods
{
	optional int32 m_groupid = 1[(yd_fieldoptions.field_cname) = "组id"];
	optional int32 m_itemID = 2[(yd_fieldoptions.field_cname) = "道具id"];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "价格类型"];
	optional int32 m_price = 4[(yd_fieldoptions.field_cname) = "价格"];
	optional int32 m_itemNum = 5[(yd_fieldoptions.field_cname) = "道具数量"];
}


message Sheet_FindtreasureActivegoods
{
	repeated E_FindtreasureActivegoods E_FindtreasureActivegoods_List = 1[(yd_fieldoptions.field_arysize)=4];
}

message E_FindtreasureActiveshopGoodDesc
{
	optional int32 m_num = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_id = 2[(yd_fieldoptions.field_cname) = "id"];
}

message E_FindtreasureActiveshop
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_groupid = 2[(yd_fieldoptions.field_cname) = "组id"];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "价格类型"];
	optional int32 m_price = 4[(yd_fieldoptions.field_cname) = "价格"];
	optional int32 m_times = 5[(yd_fieldoptions.field_cname) = "限购次数"];
	repeated E_FindtreasureActiveshopGoodDesc m_good = 6[(yd_fieldoptions.field_cname) = "商品", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_FindtreasureActiveshop
{
	repeated E_FindtreasureActiveshop E_FindtreasureActiveshop_List = 1[(yd_fieldoptions.field_arysize)=4];
}

message E_FindtreasureShop
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_groupid = 2[(yd_fieldoptions.field_cname) = "组id"];
	optional int32 m_goodsID = 3[(yd_fieldoptions.field_cname) = "商品id"];
	optional int32 m_goodsNum = 4[(yd_fieldoptions.field_cname) = "商品数量"];
	optional int32 m_costID = 5[(yd_fieldoptions.field_cname) = "货币id"];
	optional int32 m_num = 6[(yd_fieldoptions.field_cname) = "货币数量"];
}


message Sheet_FindtreasureShop
{
	repeated E_FindtreasureShop E_FindtreasureShop_List = 1[(yd_fieldoptions.field_arysize)=16];
}
